app/build.gradle

```diff
@@ -12,6 +12,8 @@ keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
 
 android {
     compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+    ndkVersion "22.0.7026061"
```

Comment:
For reproducible builds this would be nice, but for the rest it is not really useful.

Reply:
🤔 I see